.github/workflows/import-linter.yml

```
@@ -174,6 +174,23 @@ jobs:
                     # Full history required for telemetry/audit ancestor and commit-existence gates.
                     fetch-depth: 0
 
+            -   name: Fetch Git LFS objects for architecture governance
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)